client/galaxy/scripts/mvc/library/library-folderlist-view.js

```diff
@@ -271,6 +273,7 @@ var FolderListView = Backbone.View.extend({
         event.preventDefault();
         this.current_sort_order = this.current_sort_order === "asc" ? "desc" : "asc";
         this.current_sort_key = event.currentTarget.className.replace("sort-folder-", "");
+        console.log("current sort: " + this.current_sort_key)
```

Comment:
could you please remove or swap to something like `Galaxy.emit.debug("library-folderlist-view: sortColumnClicked", logstring);`